pilot/pkg/config/kube/gateway/templates/deployment.yaml

```diff
@@ -32,9 +32,40 @@ spec:
           .Labels
           | nindent 8}}
     spec:
+      {{- if .KubeVersion122 }}
```

Comment:
cc @howardjohn @Kmoneal 

Will this work for 1.23+? I thought the plan was to support 1.23 in testing after it is released and we can add it to the verified checks.

I'm wondering since above you had

```
	else if (semverCompare ">=1.22-0" .Capabilities.KubeVersion.GitVersion)
```

Reply:
This is the same, it comes from `kube.IsAtLeastVersion(d.client, 22)`. It is poorly named though